controllers/materials/helpers.js

```diff
@@ -2,7 +2,17 @@
 const { reduce } = require('lodash');
 const { check } = require('express-validator/check');
 
-const { errorTranslator } = require('../../common/validators');
+function errorTranslator(prefix) {
```

Comment:
I have deliberately inlined this because this is related to `express-validator` which we are moving away from. It shouldn't be common as that encourages reuse we want to avoid.